sqlstore/src/main/resources/db/migration/V8500__sequencing_container_models.sql

```diff
@@ -0,0 +1,84 @@
+DROP TABLE IF EXISTS SequencingContainerModel_Platform;
+DROP TABLE IF EXISTS SequencingContainerModel;
+
+CREATE TABLE SequencingContainerModel (
+  sequencingContainerModelId bigint(20) NOT NULL AUTO_INCREMENT,
+  alias varchar(255) NOT NULL,
+  identificationBarcode varchar(255),
+  partitionCount int NOT NULL,
+  platformType varchar(255) NOT NULL,
+  fallback tinyint(1) NOT NULL DEFAULT 0,
+  archived tinyint(1) NOT NULL DEFAULT 0,
+  PRIMARY KEY (sequencingContainerModelId)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE SequencingContainerModel_Platform (
+  sequencingContainerModelId bigint(20) NOT NULL,
+  platformId bigint(20) NOT NULL,
+  PRIMARY KEY (sequencingContainerModelId, platformId),
+  CONSTRAINT fk_SequencingContainerModel_Platform_model FOREIGN KEY (sequencingContainerModelId) REFERENCES SequencingContainerModel (sequencingContainerModelId),
+  CONSTRAINT fk_SequencingContainerModel_Platform_platform FOREIGN KEY (platformId) REFERENCES Platform (platformId)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- StartNoTest
+DROP PROCEDURE IF EXISTS tempMakeModels;
+
+DELIMITER //
+CREATE PROCEDURE tempMakeModels() BEGIN
+  INSERT INTO SequencingContainerModel (alias, partitionCount, platformType, fallback)
+  SELECT DISTINCT CONCAT('Generic ', ps.partitionSize, '-',
```

Comment:
What information is the "Generic " prefix intended to convey?

Reply:
Meant to mean that the exact model is not known. I'm open to suggestions for better wording. Ideally, these will be archived after more appropriate models are added. I've just now added some known Illumina flow cell models, but don't think any generic ones should be marked as archived since they can't be guaranteed to cover all potential cases.